src/main/java/Task.java

```diff
@@ -0,0 +1,39 @@
+public class Task {
+
+    private boolean isPending = true;
+    private String description;
+
+    public Task(String description) {
+        this.description = description;
+    }
+
+    public boolean isPending() {
+        return isPending;
+    }
+
+    public void setPending(boolean isPending) {
+        this.isPending = isPending;
+    }
+
+    public String getTaskDesc() {
+        return description;
+    }
+
+    public void setTaskDesc(String taskDesc) {
+        this.description = taskDesc;
+    }
+
+    public void markAsDone() {
+        this.isPending = false;
+    }
+
+    public String getType() {
+        return "[ ]";
+    }
+
+    public String getAdditionalInfo() {
+        return "";
+    }
```

Comment:
You could consider merging the two methods into one toString() method which returns the relevant string. Perhaps this would be better instead of returning an empty string. This applies to the Task class and its subclasses as well.